src/main/app/src/app/zaken/zaak-view/zaak-view.component.html

```diff
@@ -156,9 +156,9 @@
       <mat-card class="zaken-card w50 flex-1">
         <mat-card-header class="flex-row space-between">
           <mat-card-title>{{ zaak.identificatie }}</mat-card-title>
-          <mat-card-subtitle
-            >{{ zaak.zaaktype.omschrijving }}</mat-card-subtitle
-          >
+          <mat-card-subtitle>{{
+            zaak.zaaktype.omschrijving
+          }}</mat-card-subtitle>
```

Comment:
this has no change besides formatting, might be wrong

Reply:
Yess; just formatting